src/style.css

```diff
@@ -1,10 +1,12 @@
 body {
```

Comment:
- [Optoinal] Great job adding pre-request and usage on the readme.👍👍 I think it would be better to list all instruction with a description to make your README file looks professional 👍👍

👇👇👇👇sample👇👇

-- To clone this project use:
```bash

git clone repository-link

```
-- for installing dev dependecies use:

```bash

npm install

```

-- to run the project use:

```bash

npm run build

```

-- After installing the dependencies you can run tests using this command:

```bash

npm run test

```